src/pymor/models/neural_network.py

```diff
@@ -163,7 +163,7 @@ def _compute(self, quantities, data, mu):
             output = self._scale_target(output)
             if isinstance(output, torch.Tensor):
                 output = output.numpy()
-            data['output'] = output
+            data['output'] = output.reshape((self.dim_output, 1))
```

Comment:
@HenKlei, can you take a look? Apparently, `NeuralNetworkStatefreeOutputModel` always has `dim_output == 0`, causing this line to fail.